### PR TITLE
Revert "Revert VZ-2346: Update to newer Rancher"

### PR DESCRIPTION
**Pull request by kminder**
Reverts the revert of the Rancher update effectively updating Rancher to 2.5.7.

### CODE REVIEW

File: tests/e2e/verify-install/kubernetes/kubernetes_test.go [hunk 1]
```diff
@@ -137,17 +137,13 @@ var _ = ginkgo.Describe("Kubernetes Cluster",
 					gomega.Expect(vzComponentPresent(name, "cattle-system")).To(gomega.BeFalse())
 				},
 				ginkgoExt.Entry("includes rancher", "rancher", false),
-				ginkgoExt.Entry("includes rancher-agent", "cattle-node-agent", false),
```
Comment (desagar) [inline tests/e2e/verify-install/kubernetes/kubernetes_test.go:140]: is there no longer a node agent and cluster agent? What is in the managed cluster when the registration is completed?

**Comment by desagar** [comment]
another question - are the changes in the third-party/charts dir all part of the vanilla Rancher helm chart or did we have any edits on top of it which need to be documented?

**Review by desagar — APPROVED** [review]

File: tests/e2e/verify-install/kubernetes/kubernetes_test.go (hunk 1 above)
Comment (kminder) [reply tests/e2e/verify-install/kubernetes/kubernetes_test.go:140]: In the multi cluster case I believe there ends up being a rancher-agent on the managed cluster.  On the admin cluster there is no longer a separate rancher-agent.

**Comment by kminder** [comment]
In response to
> another question - are the changes in the third-party/charts dir all part of the vanilla Rancher helm chart or did we have any edits on top of it which need to be documented?
All of the helm chart changes are from the upstream Rancher.

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**